src/Microsoft.Fast.Components.FluentUI/Components/Slider/FluentSlider.razor.cs

```diff
@@ -5,25 +5,25 @@
 
 namespace Microsoft.Fast.Components.FluentUI;
 
-public partial class FluentSlider : FluentInputBase<int>
+public partial class FluentSlider<TValue> : FluentInputBase<TValue>
```

Comment:
If this `TValue` accepts only number, you should add a `where` clause:
- `where T : struct, IComparable, IComparable<T>, IConvertible, IEquatable<T>, IFormattable`
- Better since .NET7: ` where T : System.Numerics.INumber<T>`

Reply:
Solved by doing:
```
#if NET7_0_OR_GREATER
    where TValue : System.Numerics.INumber<TValue>
#else
    where TValue : struct, IComparable, IComparable<TValue>, IConvertible, IEquatable<TValue>, IFormattable
#endif
```

Reply:
Great :-)